tests/env-vars.sh

```diff
@@ -5,16 +5,16 @@ set -ex  # file is sourced so turn off at end
 # export SKYSCAN_OUTPUT_DIR=$PWD/output-dir -- rely on user value
 export SKYSCAN_BROKER_CLIENT=rabbitmq
 # note=auth env vars are in job(s)
-export EWMS_PILOT_TASK_TIMEOUT=600
+export EWMS_PILOT_TASK_TIMEOUT=1800
 # export SKYSCAN_DEBUG_DIR=debug-pkl-dir -- rely on user value
 export SKYSCAN_MQ_TIMEOUT_TO_CLIENTS=5
-# export SKYSCAN_MQ_TIMEOUT_FROM_CLIENTS=60  # use default
-export SKYSCAN_MQ_CLIENT_TIMEOUT_WAIT_FOR_FIRST_MESSAGE=120
+export SKYSCAN_MQ_TIMEOUT_FROM_CLIENTS=1800
+# export SKYSCAN_MQ_CLIENT_TIMEOUT_WAIT_FOR_FIRST_MESSAGE=120
```

Comment:
I'm fine with most of these env values. However, do we expect any reco to take 30 mins?

Reply:
hopefully not, but before it was using the default which was actually 3 days.